MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,494 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control to handle the data.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following attribute to handle the data, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.name?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text.
+</td>
+</tr>
+<tr>
+<td>
+{{'[GroupName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.groupname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the group name which groups the fields in the data form.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShortName](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.shortname?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the label text. It takes higher priority than Name.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateField](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.autogeneratefield?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies whether the field should be auto generated or not. 
+</td>
+</tr>
+<tr>
+<td>
+{{'[ResourceType](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.resourcetype?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies the Resources File (.Resx) which is used to localize the Display attribute of `Name`, `ShortName`, `GroupName`, `Prompt` and `ItemsSource` values.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Prompt](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.prompt?view=net-7.0)' |  markdownify }}
+</td>
+<td>
+Specifies watermark text for the editor.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Order](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute.order?view=net-7.0)' | markdownify }}
+</td>
+<td>
+Specifies the order of field in the data form.
+</td>
+</tr>
+</table>
+
+{% tabs %}
```

Comment:
Add on example for Display annotaion

[Display(Name= "First Name", GroupName = "Name", Prompt = "Enter your name)]

Remove all other exampls for display attribute

Reply:
above mentioned example removed